pkg/server/container_create_windows.go

```diff
@@ -358,6 +358,14 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 				return nil, errors.Errorf(`sandbox://' mounts are only supported for LCOW`, src)
 			}
 			mountType = "bind"
+		} else if strings.HasPrefix(src, "kubernetes.io~empty-dir") {
```

Comment:
Kubelet passes the path like below to containerD for emptyDir.

Mounts:[{Destination:(string)/usr/share/nginx/html Type:(string)bind Source:(string)C:\var\lib\kubelet\pods\945f981f-6d28-11e9-8c76-00155db2fa38\volumes\kubernetes.io~empty-dir\shared-data Options:([]string)[rw rbind]}] 

Though seems your check is assuming that src path starts with kubernetes.io~empty-dir.

Reply:
Is `C:\var\lib\kubelet\pods\945f981f-6d28-11e9-8c76-00155db2fa38` the root directory in this case? Is there a requirement that the contents be under the "volumes" directory? 

Right now, emptyDirs are stored in the uvm's scratch vdh under "sandboxMounts." Would this need to change?

Reply:
yes, C:\var\lib\kubelet\pods is fixed directory and 945f981f-6d28-11e9-8c76-00155db2fa38 is the pod Id.
It should not matter where emptyDirs should stored in UVM.  
I have forwarded you email that has details for this requirement.

Reply:
@katiewasnothere - Sandbox mounts in the guest is fine. If you want to make a folder called "emptydir" as well as sandbox mounts that seems fine but overkill. I think basically we strip an part of the path up to `kubernetes.io~empty-dir` and then assume the "rest" is a `sandbox://` path

Reply:
Updated.